systemtests/src/main/java/io/enmasse/systemtest/TestUtils.java

```diff
@@ -1026,6 +1026,11 @@ public static void deleteAddressSpace(AddressApiClient addressApiClient, Address
         TimeMeasuringSystem.stopOperation(operationID);
     }
 
+    public static void deleteAllAddressSpaces(AddressApiClient addressApiClient, GlobalLogCollector logCollector) throws Exception {
```

Comment:
Why is there logCollector, you never use it.

Reply:
Removed